lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -18,7 +18,12 @@ def callback(request, _uri, headers):
         data along with the data from response_overrides and dummy values for any
         additional required fields.
         """
+        is_new_thread = thread_data.pop("new_thread")
         response_data = make_minimal_cs_thread(thread_data)
+        # pop resp_total as it is not part of response from CS on thread creation
+        # request
+        if is_new_thread:
+            response_data.pop("resp_total")
```

Comment:
I don't believe it always returns `resp_total`. It looks like when it's a question, it is not returned and instead returns `non_endorsed_response_total` https://github.com/edx/cs_comments_service/blob/master/presenters/thread.rb

Seems like `resp_total` is not guaranteed.